sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ExpressionEvalHelper.scala

```diff
@@ -133,37 +169,39 @@ trait ExpressionEvalHelper extends GeneratorDrivenPropertyChecks {
     }
   }
 
+  private def evaluateWithoutCodegen(
```

Comment:
Why are we adding this method?

Reply:
Just for readability (so, reverting is ok to me).

Reply:
Why not rename the actual method?

Reply:
aha, ok. I'll fix that way. thanks.